tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/AppFilterSuite.scala

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
Nit: only 2022 should be mentioned as it's a new file.

Reply:
Resolved.